### PR TITLE
qemu.rst: update build instructions

### PR DESCRIPTION
**Pull request by jforissier**
Update build instructions for EWAOL with ledge security on QEMU.

Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

File: docs/install/qemu.rst [hunk 1]
```diff
@@ -3,32 +3,48 @@
 QEMU
 ####
 
-LEDGE Reference Platform (ledge-os-manifest) is a Linux distro based on
-Yocto/OpenEmbedded. It also uses firmware from Linaro :ref:`Trusted Substrate`
-(meta-ts). This document describes how to build and run for target QEMU.
+TRS is a Linux distro based on Yocto/OpenEmbedded, it builds on top of
```
Comment (jbech-linaro) [inline docs/install/qemu.rst:6]: As mentioned in #5 , this information should be moved to the "About" page (#4 is a bit about that).

File: docs/install/qemu.rst [hunk 2]
```diff
@@ -3,32 +3,48 @@
...
 
 Building
 ********
 
-..
-  [NEEDS_TO_BE_FIXED] - Empty section, see Rockpi4 as an example
+.. code-block:: bash
 
-TBD
+    cd
```
Comment (jbech-linaro) [inline docs/install/qemu.rst:16]: To stay consistent on various pages, please prefix shell lines with  `$ `.

**Comment by jbech-linaro** [comment]
Minor suggestions, other than that:
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

File: docs/install/qemu.rst (hunk 2 above)
Comment (jforissier) [reply docs/install/qemu.rst:16]: OK. The `$` signs make it a bit more difficult to copy and paste several lines at a time, that is the reason why I omitted them.

File: docs/install/qemu.rst (hunk 1 above)
Comment (jforissier) [reply docs/install/qemu.rst:6]: Will keep only "This document describes how to build and run for target QEMU".

**Comment by jforissier** [comment]
Updated, thanks.

File: docs/install/qemu.rst (hunk 2 above)
Comment (jbech-linaro) [reply docs/install/qemu.rst]: Yeah, it's something that I've noticed several times, but I think it brings some clarity to when you're supposed to enter something into a shell. Also, I believe most people usually we still end up copy line by line.

**Pull request merged**